### PR TITLE
[#2136] Added pointer-events none to iframes

### PR DESCRIPTION
**Pull request by mjschranz**
https://webmademovies.lighthouseapp.com/projects/65733/tickets/2136-cant-drag-new-events-over-youtubevimeo-videos

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @cadecairos received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/88e816951ea7271/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @cadecairos received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/5c190bba4f718b3/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/88e816951ea7271/output.txt

Total script time: 1.57 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/5c190bba4f718b3/output.txt

Total script time: 1.01 mins

Server is running on http://pr948.butyr.org/templates/basic/
Run the Test Suite: http://pr948.butyr.org/test/

File: src/plugin/plugin-list.js [hunk 1]
```diff
@@ -34,13 +34,27 @@ define( [ "util/dragndrop", "util/lang", "editor/editor", "text!layouts/plugin-l
 
       DragNDrop.helper( element, {
         start: function() {
-          var targets = butter.targets;
+          var targets = butter.targets,
+              mediaContainer = document.getElementById( butter.currentMedia.target ),
+              iframeVideo = mediaContainer.querySelector( "iframe" );
```
Comment (cadecairos) [inline src/plugin/plugin-list.js]: Can we try to do this on one line rather than two?

File: src/plugin/plugin-list.js [hunk 2]
```diff
@@ -34,13 +34,27 @@ define( [ "util/dragndrop", "util/lang", "editor/editor", "text!layouts/plugin-l
...
+          }
 
           for ( var i = 0, l = targets.length; i < l; ++i ) {
             targets[ i ].view.blink();
           }
         },
         stop: function() {
+
+          var mediaContainer = document.getElementById( butter.currentMedia.target ),
+              iframeVideo = mediaContainer.querySelector( "iframe" )
```
Comment (cadecairos) [inline src/plugin/plugin-list.js]: same here

Comment (cadecairos) [inline src/plugin/plugin-list.js]: also, we need to ensure that if someone interrupts a drag, the top function is called. we know it can be interrupted by right clicking during a drag.

If we miss the stop, the pointer-events attribute would be left on.

File: templates/assets/editors/editorhelper.js [hunk 3]
```diff
@@ -18,15 +18,28 @@
...
+          if ( options.start ) {
+            options.start();
+          }
+        },
         stop: function( event, ui ) {
+          if ( iframeVideo ) {
+            iframeVideo.style.pointerEvents = "auto";
+          }
+
           if ( options.end ) {
```
Comment (cadecairos) [inline templates/assets/editors/editorhelper.js:43]: Why did you add to this file?

Comment (mjschranz) [inline templates/assets/editors/editorhelper.js:43]: Because we need to cover this for both adding new plugins and moving around the containers for dragging/resizing positioning.

File: src/plugin/plugin-list.js (hunk 2 above)
Comment (mjschranz) [inline src/plugin/plugin-list.js]: How would someone interrupt a drag? The stop function is fired as soon as the person lets go of their mouse, which of course then resets the point-events back to auto.
